app/src/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./styles/index.scss";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
```

Comment:
Do we want to keep the service worker?

Reply:
So.... my job for the last ~4 years has been working on the Docs/Sheets/Slides service worker. I would really discourage using it unless we find a strong need for it (eg we need offline support or we encounter performance issues that can't be solved in a simpler way). It can be a big maintenance/debugging challenge, can cause performance issues, and has a pretty high ramp-up cost for devs to understand how it works.

TL;DR I think we should drop this

Reply:
sounds good to me @aaronsn

Reply:
sure, I will remove it.